javaws-wrapper/build.gradle

```diff
@@ -20,38 +24,21 @@ startShadowScripts {
 	applicationName = 'javaws'
 }
 
-task doc(type: Exec) {
-	def outputDir = file('build/doc')
-	outputs.dir outputDir
-
-	doFirst {
-		mkdir outputDir
-	}
-
-	commandLine 'pandoc', 'README.md', '-o', "$outputDir/README.pdf"
-}
-
 distributions {
 	shadow {
 		baseName = 'javaws'
 		contents {
-			from(doc) {
-				include '**'
+			from(readmeToPdf) {
```

Comment:
we should name and place the documentation consistently with the agent. e.g. also under documentation/userguide.md

Reply:
i've renamed all documentation files now README.md (this way GitHub shows the file when opening a subfolder. On build these are renamed to userguide.